packages/agoric-cli/src/types.d.ts

```diff
@@ -0,0 +1,7 @@
+// This package has maxNodeModuleJsDepth=0 to prevent ambient imports.
+// A consequence is that types that are only provided ambiently have to be defined.
+
+export type Brand = unknown;
+export type Amount = { brand: Brand; value: bigint };
+export type Instance = unknown;
+export type Proposal = any;
```

Comment:
Why are these being left untyped?

Reply:
Explained in a comment

Reply:
you mean the line above that says
> types that are only provided ambiently have to be defined.

Why is it better to declare them as `unknown` or `any` than to copy their actual types?  Does the CLI not rely on their values?

Reply:
> Does the CLI not rely on their values?

Well the runtime doesn't rely on these typedefs at all. This package doesn't export anything to consumes so the types are there for dx productivity. The trade-off here is that there's more productivity to be gained by leaving them as is (vs spending more time on this).

> Why is it better to declare them as unknown or any than to copy their actual types?

The runtime types also don't match because the CLI gets its values from RPC and doesn't have CapTP to translate them into regular objects. That's what the `xxx RpcRemote` comments are about.